src/_gettsim/transfers/elterngeld.py

```diff
@@ -384,106 +293,181 @@ def elterngeld_anteil_eink_erlass(
+    _elterngeld_anz_mehrlinge_fg: int, elterngeld_params: dict
 ) -> float:
-    """Calculate the bonus for siblings.
-
-    According to § 2a parents of siblings get a bonus.
+    """Elterngeld bonus for multiples.
 
     Parameters
     ----------
-    elterngeld_eink_erlass_m
-        See :func:`elterngeld_eink_erlass_m`.
-    elterngeld_geschw_bonus_anspruch
-        See :func:`elterngeld_geschw_bonus_anspruch`.
+    _elterngeld_anz_mehrlinge_fg
+        See :func:`_elterngeld_anz_mehrlinge_fg`.
     elterngeld_params
         See params documentation :ref:`elterngeld_params <elterngeld_params>`.
 
     Returns
     -------
 
     """
-    if elterngeld_geschw_bonus_anspruch:
-        out = max(
-            elterngeld_params["geschw_bonus_aufschlag"] * elterngeld_eink_erlass_m,
-            elterngeld_params["geschw_bonus_minimum"],
-        )
-    else:
-        out = 0.0
-    return out
+    return _elterngeld_anz_mehrlinge_fg * elterngeld_params["mehrlingbonus"]
 
 
-def elterngeld_mehrlinge_bonus_m(
-    _elterngeld_anz_mehrlinge_anspruch: int, elterngeld_params: dict
-) -> float:
-    """Calculate the bonus for multiples.
+def geschwisterbonus_anspruchsberechtigt_fg(
+    anz_kinder_bis_2_fg: int,
+    anz_kinder_bis_5_fg: int,
+    elterngeld_params: dict,
+) -> bool:
+    """Siblings that give rise to Elterngeld siblings bonus.
 
     Parameters
     ----------
-    _elterngeld_anz_mehrlinge_anspruch
-        See :func:`_elterngeld_anz_mehrlinge_anspruch`.
+    alter
+        See basic input variable :ref:`alter <alter>`.
     elterngeld_params
         See params documentation :ref:`elterngeld_params <elterngeld_params>`.
 
     Returns
     -------
 
     """
-    return float(
-        _elterngeld_anz_mehrlinge_anspruch * elterngeld_params["mehrlingbonus"]
+    geschwister_unter_3 = (
+        anz_kinder_bis_2_fg >= elterngeld_params["geschwisterbonus_altersgrenzen"][3]
+    )
+    geschwister_unter_6 = (
+        anz_kinder_bis_5_fg >= elterngeld_params["geschwisterbonus_altersgrenzen"][6]
     )
 
+    return geschwister_unter_3 or geschwister_unter_6
+
+
+def _elterngeld_anz_mehrlinge_fg(
+    anz_mehrlinge_jüngstes_kind_fg: int,
+) -> int:
+    """Number of multiples of the youngest child.
+
+    Parameters
+    ----------
+    anz_mehrlinge_jüngstes_kind_fg
+        See :func:`anz_mehrlinge_jüngstes_kind_fg`.
+
+    Returns
+    -------
+
+    """
+    out = anz_mehrlinge_jüngstes_kind_fg - 1
+    return max(out, 0)
+
+
+def elterngeld_anrechenbares_nettoeinkommen_m(
+    bruttolohn_m: float,
+    lohnst_m: float,
+    soli_st_lohnst_m: float,
+) -> float:
+    """Income that reduces the Elterngeld claim.
+
+    Parameters
+    ----------
+    bruttolohn_m
+        See basic input variable :ref:`bruttolohn_m <bruttolohn_m>`.
+    lohnst_m
+        See :func:`lohnst_m`.
+    soli_st_lohnst_m
+        See :func:`soli_st_lohnst_m`.
+
+    Returns
+    -------
+
+    """
+    return bruttolohn_m - lohnst_m - soli_st_lohnst_m
```

Comment:
`lohnst_m` should be calculated without taking into account adaptions to the care insurance rate (having no child and being older than 23, living in Saxony). But I guess that is just impossible to incorprate?

"wobei die Höhe der Teilbeträge ohne Berücksichtigung der besonderen Regelungen zur Berechnung der Beiträge nach [§ 55 Absatz 3](https://www.buzer.de/55_SGB_11.htm) und [§ 58 Absatz 3 des Elften Buches Sozialgesetzbuch](https://www.buzer.de/58_SGB_11.htm) bestimmt wird."

Reply:
Definitely not impossible. We would need to copy the functions and change the inputs of roundabout 10 functions of the Lohnsteuer module.

@hmgaudecker Given the other imprecisions in this module, do you think this is worth it? Maybe we should the new interface can provide a fancy way of handling this? Sounds very similar to the Bedarfsgemeinschaften issue to me.

Reply:
Yes, rather keep it simple here. Just make a note on this!